theme.css

```
@@ -1841,6 +1852,10 @@ ul.comparison-list > li em {
   border-color: var(--white-6);
 }
 
+.capped-card-content {
+  background: var(--white-1);
```

Looks like white-2 matches with the white currently used in the card.

```suggestion
  background-color: var(--white-2);
```